ci/build/build-code-server.sh

```diff
@@ -3,9 +3,6 @@ set -euo pipefail
 
 # Builds code-server into out and the frontend into dist.
 
-# MINIFY controls whether parcel minifies dist.
-MINIFY=${MINIFY-true}
```

Comment:
Should we leave the ability to disable minification? Implementation could look something like:

```bash
local opts=()
[[ $MINIFY ]] && opts+=("-p" "tinyfy")

yarn browserify "${opts[@]}" src/browser/register.ts -o out/browser/register.browserified.js
```

Reply:
Honestly, I didn't know how I might implement this so I removed it 😅 Thanks for writing up the implementation and making it easy to copy/paste!